src/profile-logic/profile-data.js

```diff
@@ -818,6 +818,9 @@ export function invertCallstack(thread: Thread): Thread {
     const prefixAndFrameToStack = new Map();
     const frameCount = frameTable.length;
 
+    // Returns the stackIndex for a specific frame (that is, a function and its
+    // context), and a specific prefix. If it's not existing yet it will create
+    // a new stack entry and returns its index.
```

Comment:
"and return its index", no s after "return", because the "will" applies to this part of the sentence as well